### ISSUE TITLE
[fvt]2.13.5:openbmc :return value for rpower node (invalid node)  wrong

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#3002]
env:openbmc
build:lsxcatd -v
Version 2.13.4 (git commit 6331ada77447f9eaaf4e6b71224f38d4d6643eff, built Sat May  6 06:15:59 EDT 2017)
How to reproduce:
```
1.
[root@c910f03c11k03 ~]# rpower testnode stat
This openbmc related function is unsupported and disabled. To bypass, run the following: 
	export XCAT_OPENBMC_DEVEL=YES
[root@c910f03c11k03 ~]# XCAT_OPENBMC_DEVEL=YES rpower testnode stat
Warning: Currently running development code, use at your own risk.  Unset XCAT_OPENBMC_DEVEL to disable.
testnode: Unable to get information from openbmc table
2.
[root@c910f03c11k03 ~]# echo $?
0
3.
[root@c910f03c11k03 ~]# lsdef testnode
Object name: testnode
    groups=all
    mgt=openbmc
    postbootscripts=otherpkgs
    postscripts=syslog,remoteshell,syncfiles
[root@c910f03c11k03 ~]# 

```

**Comment by xuweibj** [comment]
The issue was fixed in pull request #3016

**Comment by junxiawang** [comment]
I have verified
